modules/xfem/include/userobjects/MeshCut2DRankTwoTensorNucleation.h

```diff
@@ -34,13 +40,22 @@ class MeshCut2DRankTwoTensorNucleation : public MeshCut2DNucleationBase
   const Point _point1;
   const Point _point2;
 
-  /// Length of crack to be nucleated
-  const Real _nucleation_length;
-
   /// Transformed Jacobian weights
   const MooseArray<Real> & _JxW;
   const MooseArray<Real> & _coord;
 
   virtual bool
   doesElementCrack(std::pair<RealVectorValue, RealVectorValue> & cutterElemNodes) override;
+
+  // FIXME Lynn Copy from TraceRayTools.C in rayTracing module.  Remove once this function migrates
+  // to libmesh.
+  /// The standard "looser" tolerance to use in ray tracing when having difficulty finding intersection
```

Comment:
Since `TRACE_TOLERANCE` is only used inside this function, it seems better to make it a local variable in that function.